quipucords/tests/factories.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2022  Red Hat, Inc.
+
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+
+"""factories to help testing Quipucords."""
+import random
+
+import factory
+from factory.django import DjangoModelFactory
+
+
+class SystemFingerprintFactory(DjangoModelFactory):
+    """SystemFingerprint factory."""
+
+    name = factory.Faker("hostname")
+    os_name = "Red Hat Enterprise Linux"
+    ip_addresses = factory.List([factory.Faker("ipv4")])
+    architecture = factory.Iterator(["x86_64", "ARM"])
+
+    class Meta:
+        """Factory options."""
+
+        model = "api.SystemFingerprint"
+
+
+class DeploymentReportFactory(DjangoModelFactory):
```

Comment:
For consistency, maybe this factory should be named `DeploymentsReportFactory` since it's a factory for `DeploymentsReport`

Reply:
In that case, I agree that it's safer to leave it as is.